packages/react-components/react-drawer/src/components/Drawer/useDrawerStyles.styles.ts

```diff
@@ -12,24 +12,39 @@ export const drawerClassNames: SlotClassNames<DrawerSlots> = {
  */
 export const drawerCSSVars = {
   size: '--fui-Drawer--size',
+  borderRadius: '--fui-Drawer--borderRadius',
```

Comment:
the CSS variables here are only used in this file, I can't really see any reason why it's necessary instead of just writing the CSS directly in the makeStyle calls

Reply:
It makes sense. But as this isn't part of this PR's scope, it'll be fixed on another PR. The work can be tracked here:
https://github.com/marcosmoura/fluentui/tree/feat/split-drawer/packages/react-components/react-drawer/src